docs/man/skuba-auth-login.1.md

```diff
@@ -19,7 +19,7 @@ login - Authenticate to a cluster and authorized with kubeconfig
   Print usage statement.
 
 **--server, -s**
-  (Required) The OIDC dex server url (https://<controller plane IP/FQDN>:<port>)
+  (Required) The OIDC dex server url https://<IP/FQDN>:<Port> (specify port 32000 for standard CaaSP deployments)
```

Comment:
For Caasp4 deployment, Dex port 32000 is required.

Reply:
This would confuse user that port always 32000.